src/Components/GrowthChart/GrowthChart.tsx

```diff
@@ -26,6 +27,9 @@ export const GrowthChart = () => {
 
     const yAxisValues = { minDataValue, maxDataValue };
 
+    const optionsObject: { annotations: AnnotationLabel[] } = { annotations: [] };
```

Comment:
Provide a more descriptive name

Reply:
can we keep it as annotations? 

Later we will make it work for months aswell, instead of making a new hook, we can just use "GrowthChartAnnotations", but we send with metadata that says months/weeks. 
If(months){yearannotations}
If(weeks){WeekAnnotations}

What do you think?

Reply:
That sounds good, @petterav!